sandbox/auth/session/idclaims.go

```diff
@@ -0,0 +1,63 @@
+package session
+
+import "github.com/go-jose/go-jose/v3/jwt"
+
+// Standard claims:
+// oidc: https://openid.net/specs/openid-connect-basic-1_0.html#IDToken
+// jwt: https://datatracker.ietf.org/doc/html/rfc7519#section-4
+// oidc user: https://openid.net/specs/openid-connect-basic-1_0.html#StandardClaims
+
+type IDClaims struct {
+	ID string `json:"jti,omitempty"`
+
+	Issuer    string           `json:"iss,omitempty"`
+	Subject   string           `json:"sub,omitempty"`
+	Audience  jwt.Audience     `json:"aud,omitempty"`
+	Expiry    *jwt.NumericDate `json:"exp,omitempty"`
+	NotBefore *jwt.NumericDate `json:"nbf,omitempty"`
+	IssuedAt  *jwt.NumericDate `json:"iat,omitempty"`
+
+	Nonce           string `json:"nonce,omitempty"`
+	AccessTokenHash string `json:"at_hash,omitempty"`
+
+	// Email scope:
+
+	Email         string `json:"email,omitempty"`
+	EmailVerified bool   `json:"email_verified,omitempty"`
+
+	// Profile scope:
+
+	Name       string           `json:"name,omitempty"`
+	GivenName  string           `json:"given_name,omitempty"`
+	FamilyName string           `json:"family_name,omitempty"`
+	Username   string           `json:"username,omitempty"`
+	Website    string           `json:"website,omitempty"`
```

Comment:
What's this? Personal website?

Reply:
Yeah, the website of the user. Although, maybe I'll get rid of this and of username, since we're not really using them. What do you think?

Reply:
Looking at the standard again, if I keep username, it should be`preferred_username`